### PR TITLE
Fix: fixed test file paths

### PR DESCRIPTION
**Pull request by Sean1572**
Fixes test file paths. Please note I will squash this PR

### CODE REVIEW

File: pyha_tests/test_pyha.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 '''
 from pathlib import Path
 import importlib
+import os
```
Comment (ntlhui) [inline pyha_tests/test_pyha.py:5]: Extraneous import

File: .github/workflows/env_test.yml [hunk 2]
```diff
@@ -28,4 +28,4 @@ jobs:
         env:
           NAS_CREDS: ${{ secrets.NAS_CREDS }}
         run: |
-          poetry run python -m pytest pyha_tests
\ No newline at end of file
+          poetry run python -m pytest pyha_tests -rP
```
Comment (ntlhui) [inline .github/workflows/env_test.yml:31]: What is this flag for?

**Review by ntlhui — CHANGES_REQUESTED** [review]

File: .github/workflows/env_test.yml (hunk 2 above)
Comment (Sean1572) [reply .github/workflows/env_test.yml:31]: collects and prints stdout from test after the test ends. IE log/print statements

Comment (ntlhui) [reply .github/workflows/env_test.yml:31]: Sounds good, probably good to put for other repos.  Can you add to python-repo-example?

**Review by ntlhui — APPROVED** [review]

**Pull request merged**